lib/menu.rb

```diff
@@ -0,0 +1,35 @@
+require 'csv'
+require_relative 'dish'
+
+class Menu
+
+  attr_accessor :dish_list
+
+  def initialize(dish_list = [])
+    @dish_list = dish_list
+    load_file_with_dishes
+  end
+
+  def load_file_with_dishes
```

Comment:
At some point it would good to consider using a class that deals with parsing CSV data and returning ruby objects that you want, so that your menu class can use the behaviour but not be responsible for file reading and writing